boring/src/ssl/mod.rs

```diff
@@ -834,6 +834,47 @@ pub fn select_next_proto<'a>(server: &[u8], client: &'a [u8]) -> Option<&'a [u8]
     }
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub struct Ssl3AlertLevel(i32);
+
+impl Ssl3AlertLevel {
```

Comment:
This does not seem to follow the idiom used by the various ints provided by boring. The others have `Self` constants that wrap the `ffi` values.